content/hardware/04.pro/boards/portenta-c33/tutorials/user-manual/content.md

```diff
@@ -1426,6 +1426,127 @@ Here's what each section of the example code does:
+
+![Arduino IoT Cloud Devices page](assets/user-manual-14.png) 
+
+The **Setup Device** pop-up window will appear, navigate into **AUTOMATIC** and select the **Arduino board** option:
+
+![Arduino IoT Cloud Setup Device pop-up window](assets/user-manual-15.png) 
+
+After a while, your Portenta C33 board should be discovered by the Arduino IoT Cloud, as shown below: 
+
+![Arduino IoT Cloud Setup Device pop-up window](assets/user-manual-16.png) 
+
+Click the **CONFIGURE** button, give your board a name, and select the connection. In this example, we will use a Wi-Fi® connection; you can also use an Ethernet connection with a [Portenta Max Carrier](https://store.arduino.cc/products/portenta-max-carrier), a [Portenta Breakout](https://store.arduino.cc/products/arduino-portenta-breakout), or a custom-made board with an Ethernet connector. Your Portenta C33 board will be configured to communicate securely with the Arduino IoT Cloud. This process can take a while. 
+
+![Arduino IoT Cloud Setup Device pop-up window](assets/user-manual-17.png) 
+
+Now that you have your Portenta C33 and your Arduino IoT Cloud configured let's create a "Thing" to test the connection between your board and the Arduino IoT Cloud. Navigate into **Things** and select the **CREATE THING** button; give your thing a name. 
+
+![Arduino IoT Cloud "Thing" setup](assets/user-manual-18.png) 
+
+Then, navigate into **Associate Device** and click the **Select Device** button, select your Portenta C33 board and associate it to your "Thing". Then, navigate into **Network** and click the **Configure** button; enter your network credentials. Now you are ready to add some variables to your "Thing"; navigate into **Cloud Variables** and click the **ADD VARIABLE** button. 
+
+![Arduino IoT Cloud "Thing" setup](assets/user-manual-19.png) 
+
+Add one variable with the following characteristics:
+
+- **Name**: `led`
+- **Variable type**: `boolean`
+- **Variable permission** `Read & Write`
+- **Variable update policy**: `On change`
+
+You should see the `led` variable on the **Cloud Variables** section. Now, navigate into **Dashboards** and select the **BUILD DASHBOARD** button, this will create a new dashboard; give your dashboard a name. Now, add the following to your widgets to your dashboard:
+
+- **Switch**: name the widget `Switch` and link it to the `led` variable you created before. 
+- **LED**: name the widget `led` and link it to the `led` variable you created before. 
+
+Your dashboard should see like the following:
+
+![Arduino IoT Cloud "Thing" setup](assets/user-manual-20.png)
+
+Now, go back to your **Things** and open the "Thing" you just have created. In the "Thing" setup page, navigate into **Sketch**, you should see now an online editor. 
+
+In the generated sketch, you just need to define the `led` pin as an output:
+
+```arduino
+// LED_BUILTIN macro access the onboard green LED 
+pinMode(LED_BUILTIN, OUTPUT);  
+```

```

Comment:
I know it is obvious, but some of the most newbies will have problems to understand where to put these lines, please improve it put in the more code or text to explain that you have to put that on the setup

Reply:
I agree; I put the whole setup() function so people stay aware of the code.